### PR TITLE
Feature/validation set

### PR DESCRIPTION
**Pull request by leoTiez**

### CODE REVIEW

File: RCNN_tf.py [hunk 1]
```diff
@@ -579,7 +582,21 @@ def train(self, train_data_feats, train_data_labels, val_data_feats,
 
                 print "\nEpoch:", (epoch + 1), \
                     "cost =", "{:.3f}".format(avg_cost), \
-                    "test accuracy: {:.3f}".format(val_acc)
+                    "validation accuracy: {:.3f}".format(val_acc)
```
Comment (adrianc-a) [inline RCNN_tf.py:585]: Cool. I think it would make more sense to have the validation loss plotted with the training loss. I think i can ad this later this though. Should just be `sess.run([..., self.cross_entropy]...)` right?

**Review by adrianc-a — APPROVED** [review]
